### ISSUE TITLE
Flexbox

### ISSUE THREAD

**Issue by Yka59** [adobe/brackets#14658]
### Prerequisites

* [ ] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
* [ ] Did you perform a cursory search to see if your bug or enhancement is already reported?
* [ ] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md)

### Description

[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

### Versions

Please include the OS and what version of the OS you're running.
Please include the version of Brackets. You can find it under `Help -> About Brackets` (Windows and Linux) or `Brackets -> About Brackets` (macOS)

**Comment by Yka59** [comment]
I've just downloaded Brackets, and quit new to coding. I'm trying to work with Flexbox in Brackets but for some reason It don't work? C attach file. Is there an extension, or what should I do?
![nayttokuva 2019-2-25 kello 15 25 00](https://user-images.githubusercontent.com/47443749/53341201-9b0d1b00-3913-11e9-8b9f-f42ee678e748.png)

**Comment by Yka59** [comment]
Sorry almost forgot. OS 10.13.6, Bracket

**Comment by mslourens** [comment]
You are using the old flexbox spec:

- `display: box` should be `display: flex`
- `box-flex` should be just `flex`
- `box-orient` should be `flex-direction`
- `box-ordinal-group` should be `order`

So this doesn't seem to be a Brackets specific issue

**Comment by Yka59** [comment]
Hi. Thanks for your reply. My mistake, I took a book that I thought was
released 2018, but after your e-mail I took a closer look, and It was 2015.
Sorry about that, and thank you, I really appreciate It.

Brgds:Ykä!!

Yrjö Halinen
gsm: +358 40 171 4929
e-mail: yhalinen@gmail.com



ke 27. helmik. 2019 klo 21.38 Maurits Lourens (notifications@github.com)
kirjoitti:

**Comment by CLantigua2** [comment]
yes, as @mslourens stated this is due to the older version of flexbox. Current version requires you to set your display: flex on the parent container. Here is a fantastic guide for flexbox properties that should be easy to follow. [FlexBox Guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) You can also checkout this interactive flexbox froggy game to get a more hands on approach to flexbox syntax. [FlexBox Froggy](https://flexboxfroggy.com/)
